tests/management/tenant/test_model.py

```diff
@@ -386,4 +449,17 @@ def test_bulk_adding_updating_users(self):
                     subject("rbac", "role", admin_default_policy.uuid),
                 )
             ),
+            f"Expected admin default role binding to have admin default role for tenant {org_id}",
         )
+        return tenant, mapping, root, default
+
+    @patch("management.management.commands.utils.BOOT_STRAP_SERVICE")
+    def test_retrying_bulk(self, mock_bss):
```

Comment:
Maybe this should be moved to a test class for the import_tenant_user_data module specifically? We should probably add some other tests specifically at that layer too. At least just a sanity check.

Reply:
Updated